src/pages/Home/sections/LiqiuditySection/components/PoolCardContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { t } from "@lingui/macro";
+
+import { useGoToPools } from "pages/Home/hooks/useGoToPools";
+
+import glvCoin from "img/bg_coin_glv.png";
+import gmCoin from "img/bg_coin_gm.png";
+import gmxCoin from "img/bg_coin_gmx.png";
+import IcGlvPool from "img/ic_glv_pool.svg?react";
+import IcGmPool from "img/ic_gm_pool.svg?react";
+import IcGmxPool from "img/ic_gmx_pool.svg?react";
+
+import { PoolCard } from "./PoolCard";
+
+type Pool = "GM" | "GLV" | "GMX";
+
+type Props = {
+  pool: Pool;
+};
+
+const DESCRIPTIONS: Record<Pool, string> = {
+  GMX: t`Stake for rewards and governance rights`,
```

Comment:
oh, these translations won't work, since (t) should be called in render lifecycle